### ISSUE TITLE
Buttons in table and cell properties panels are missing aria-pressed attribute

### ISSUE THREAD

**Issue by FilipTokarski** [ckeditor/ckeditor5#17722]
## 📝 Provide detailed reproduction steps (if any)

When pressing buttons in table and cell properties panel, for example:

![Image](https://github.com/user-attachments/assets/455a0ee7-a29e-4581-9f33-761e3ff2872b)
![Image](https://github.com/user-attachments/assets/0057c99a-6639-4c08-8da3-ff729c69fb57)

buttons should use `aria-pressed` attribute.

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.
